values.yaml

```diff
@@ -218,11 +218,31 @@ server:
     enabled: true
     # If you need to use a http path instead of the default exec
     # path: /v1/sys/health?standbyok=true
+    
+    # When a probe fails, Kubernetes will try failureThreshold times before giving up
+    failureThreshold: 2
+    # Number of seconds after the container has started before probe initiates
+    initialDelaySeconds: 5
+    # How often (in seconds) to perform the probe
+    periodSeconds: 3
+    # Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+    # Number of seconds after which the probe times out.
+    timeoutSeconds: 5
```

Comment:
I see these were the previous defaults, but I'm wondering if the `timeoutSeconds` should be less than the `periodSeconds`? Seems like you'd want to give a probe the chance to timeout before firing another one? Same question applies to the livessProbe section.

Reply:
Good point, I switched them!